src/users/users_eods.service.ts

```diff
@@ -38,12 +38,8 @@ export default class UsersEodsService {
     return result;
   }
 
+  @Cron(CronExpression.EVERY_DAY_AT_MIDNIGHT)
   scheduleArchivingEods() {
-    this.logger.log('Schedule archiving EOD for all users');
-    if (this.isArchivingEodsScheduled) return;
```

Comment:
I think this property can be removed since it's now redundant. Same for the one in `system.service`.